video/mp_image.h

```diff
@@ -121,8 +121,16 @@ typedef struct mp_image {
     struct AVBufferRef *icc_profile;
     // Closed captions packet, if any (only after decoder)
     struct AVBufferRef *a53_cc;
+    // Other side data we don't care about.
+    struct mp_ff_side_data *ff_side_data;
+    int num_ff_side_data;
 } mp_image_t;
 
+struct mp_ff_side_data {
+    int type;
```

Comment:
Not a nack, but thinking if this should be enum AVFrameSideDataType for the shits and giggles. Internally enums are ints as far as I know so the type itself is a-OK.

Reply:
Not doing this because it would require to include ffmpeg headers in this header.

Reply:
Alright. Fine by me. I just saw the AVBufferRef underneath and thought I'd comment :) .

Reply:
The AVBufferRef works because it's an opaque struct. Unfortunately that doesn't work for enums.